op-scim-bridge/values.yaml

```diff
@@ -20,15 +23,15 @@ scim:
     # storageClass for the volume. do-block-storage is recommended for Digital Ocean. Set to "-" to set storageClass to "".
     # storageClass:
   # credentialsSecret configures the SCIM bridge to access the credentials via a secret
-  # credentialsSecret:
+  credentialsSecret:
+    # enables the `credentialsSecret`. Defaults to `true`. Either `credentialsSecret` or `cedentialsVolume` must be specified, but not both
+    enabled: true
     # name of the secret
-    # name: "{{ .Chart.Name }}-bridge-credentials"
+    name: "{{ .Chart.Name }}-bridge-credentials"
     # key of the secret
-    # key: scimsession
-    # value_json is the JSON contents of the scimsession file
-    # value_json: "{}"
-    # value_base64 is the base64 encoded contents of the scimsession file
-    # value_base64: "base64 encoded scimsession file"
```

Comment:
Unfortunately we will need to keep these as we wish to maintain support for both JSON and Base64 encoded scimsessions.

Reply:
See my above comments